app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ) -> None:
```

Comment:
```suggestion
    def __init__(
         self,
         comfort_class: int,
         clean_mark: int,
         brand: str
) -> None:
```
keep the one indent style

Reply:
the same in other similar cases

Reply:
Thank you. I remember
Done!